.tav.yml

```diff
@@ -41,26 +41,57 @@ graphql:
 express:
   versions: '>=4'
   commands: node test/instrumentation/modules/express.js
+
 express-graphql-1:
   name: express-graphql
   peerDependencies: graphql@^0.8.2
   versions: '0.6.1'
   commands: node test/instrumentation/modules/express-graphql.js
+
 express-graphql-2:
   name: express-graphql
   peerDependencies: graphql@^0.9.0
   versions: '>=0.6.2 <0.6.6'
   commands: node test/instrumentation/modules/express-graphql.js
+
 express-graphql-3:
   name: express-graphql
   peerDependencies: graphql@^0.10.0
   versions: '>=0.6.6 <0.6.8'
   commands: node test/instrumentation/modules/express-graphql.js
-express-graphql-4:
+
+express-graphql-0.6.11_10:
+  name: express-graphql
+  peerDependencies: graphql@^0.10.0
+  versions: '0.6.11'
+  commands: node test/instrumentation/modules/express-graphql.js
+express-graphql-0.6.11_11:
   name: express-graphql
   peerDependencies: graphql@^0.11.0
-  versions: '>=0.6.8'
+  versions: '0.6.11'
+  commands: node test/instrumentation/modules/express-graphql.js
+
+express-graphql-0.6.12_10:
+  name: express-graphql
+  peerDependencies: graphql@^0.10.0
+  versions: '>=0.6.12'
+  commands: node test/instrumentation/modules/express-graphql.js
+express-graphql-0.6.12_11:
+  name: express-graphql
+  peerDependencies: graphql@^0.11.0
+  versions: '>=0.6.12'
+  commands: node test/instrumentation/modules/express-graphql.js
+express-graphql-0.6.12_12:
+  name: express-graphql
+  peerDependencies: graphql@^0.12.0
+  versions: '>=0.6.12'
+  commands: node test/instrumentation/modules/express-graphql.js
+express-graphql-0.6.12_13:
+  name: express-graphql
+  peerDependencies: graphql@^0.13.0
+  versions: '>=0.6.12'
   commands: node test/instrumentation/modules/express-graphql.js
+
```

Comment:
Needing to manually describe all this multi-dimensional testing for each patch release is not ideal. We should probably see if we can figure out a way to express multi-dimensionality better in tav.

Reply:
yeah, a matrix config option would be better. Haven't had time to build that yet and the close relationship between `graphql` and `express-graphql` is the only place we have this need, so it hasn't been a big issue yet. I used to actually just test against the latest version of graphql that the given version express-graphql supported.